data/group_stage.json

```diff
@@ -0,0 +1,338 @@
+{
+  "A": {
+    "Qatar": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 5
+    },
+    "Ecuador": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 5
+    },
+    "Senegal": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 5
+    },
+    "Netherlands": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 4
+    }
+  },
+  "B": {
+    "England": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 0
+    },
+    "Iran": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 0
+    },
+    "United_States": {
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 0
+    },
+    "Unknown": {
```

Comment:
Replace **Unknown** teams (Unknown_B, Unknown_D and Unknown_E) as per comment above.